src/twisted/trial/util.py

```diff
@@ -387,3 +387,12 @@ def _listToPhrase(things, finalDelimiter, delimiter=", "):
             finalDelimiter,
             strThings[-1],
         )
+
+
+def openTestLog(path: FilePath) -> TextIO:
```

Comment:
Should this be a public API? Just asking?

Very minor comment.

Maybe make it private and give it a generic name, so that it can also be used for non log files... if needed.

```suggestion
def _openUTF8Append(path: FilePath) -> TextIO:
```

Reply:
Maybe `twisted.trial.util` should be deprecated and replaced by `twisted.trial._util`.  The current module docstring effectively says it is all meant to be private.

Reply:
Ok. Just merge. don't worry. Thanks! We can deprecated the whole `util` later.